stack/apps/push7_button/filesystem/humidity_file.c

```diff
@@ -132,29 +137,29 @@ static void execute_measurement()
 
 void humidity_file_set_measure_state(bool enable)
 {
-    timer_cancel_task(&execute_measurement);
+    timer_cancel_task(&humidity_file_execute_measurement);
     humidity_file_transmit_state = enable;
     humidity_config_file_transmit_state = enable;
 
     if (humidity_config_file_cached.enabled && humidity_file_transmit_state)
-        timer_post_task_delay(&execute_measurement, humidity_config_file_cached.interval * TIMER_TICKS_PER_SEC);
+        timer_post_task_delay(&humidity_file_execute_measurement, humidity_config_file_cached.interval * TIMER_TICKS_PER_SEC);
 }
 
 void humidity_file_set_test_mode(bool enable)
 {
     if (test_mode_state == enable)
         return;
     test_mode_state == enable;
-    timer_cancel_task(&execute_measurement);
+    timer_cancel_task(&humidity_file_execute_measurement);
     if (enable) {
         humidity_config_file_cached.interval = TESTMODE_HUMID_INTERVAL_SEC;
         humidity_config_file_cached.enabled = true;
-        timer_post_task_delay(&execute_measurement, humidity_config_file_cached.interval * TIMER_TICKS_PER_SEC);
+        timer_post_task_delay(&humidity_file_execute_measurement, humidity_config_file_cached.interval * TIMER_TICKS_PER_SEC);
```

Comment:
didn't we agree testmode would now send data of a sensor upon button press? or is it not yet included here?

Reply:
It is, see test_state_input_event_handler